src/API/WesternStatesWater.WestDaat.Tests.ManagerTests/SystemManagerTests.cs

```diff
@@ -11,6 +11,8 @@ public class SystemManagerTests : ManagerTestBase
         private readonly Mock<ILocationEngine> _locationEngineMock = new(MockBehavior.Strict);
 
         private readonly Mock<ISystemAccessor> _systemAccessorMock = new(MockBehavior.Strict);
```

Comment:
I know you didn't put these in here but we shouldn't have readonly mocks. They should be newed up for every test